python/config_settings/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//python:versions.bzl", "TOOL_VERSIONS")
+load(":config_settings.bzl", "construct_config_settings")
+
+filegroup(
+    name = "distribution",
```

Comment:
What are these distribution targets?

Reply:
Used to collect sources for bazel-in-bazel integration tests.